vulture/core.py

```diff
@@ -48,8 +47,17 @@ def _match(name, patterns, case=True):
 
 def _is_test_file(filename):
     return _match(
-        os.path.abspath(filename),
-        ["*/test/*", "*/tests/*", "*/test*.py", "*/*_test.py", "*/*-test.py"],
+        filename.resolve(),
+        [
+            "*/test/*",
```

Comment:
I added more patterns here to fix some windows errors. pathlib works differently in unix and windows it seems.